reflred/footprint.py

```diff
@@ -220,7 +227,9 @@ def _abinitio_footprint(slit1, slit2, theta, width, offset=0.):
     d2, s2x, s2y = slit2
 
     # for some reason, distances are stored in nexus files with float32 precision,
-    # causing roundoff errors in this calculation
+    # causing noticeable roundoff differences between different orders of operation
+    # (which affects the reproducibility tests, when the algorithm is altered.)
```

Comment:
Still going to have numerical differences in float64. It's a question of the precision used for the tests (~1e-8 rather than 1e-14, or maybe 1e-7 vs. 1e-13 if the calculation is unstable).

Reply:
We're currently doing a text diff on real output files to determine pass/fail, so it would take work to modify the tests, but I agree that the tests could be more carefully tuned.

Reply:
I see this is just a comment on the current state of affairs, not a change in the code. I'll be quiet now and leave well enough alone.